### PR TITLE
Git flow

### PR DESCRIPTION
**Pull request by earthpyy**

### CODE REVIEW

File: devops/git-flow/README.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+# Git Flow
+
+In this standard, we suggest you multiple implementations of Git Flow. Each flow is different. Some is better when you're developing, some is better when your project was already go live. This standard will help you pick the right one for your project.
+
+Before we start, one thing that's constant is we will use only _one branch_ as a standard.
+
+## TL;DR
+
```
Comment (piyapan039285) [inline devops/git-flow/README.md:8]: in TL;DR, I think we should add short description on each git flow. For example

* **Centralized Workflow**: have only master branch and all changes are committed into master.
* **Feature Branch**: have master and feature branch.
* **Trunk-base Development**: it is like feature branch but feature branch is very short-lived and will merge to master as fast as possible. Project should implement feature flags to turn on/off feature quickly if bugs occurred.

**Review by piyapan039285 — CHANGES_REQUESTED** [review]

File: devops/git-flow/README.md (hunk 1 above)
Comment (earthpyy) [reply devops/git-flow/README.md:8]: Updated krubbb

**Review by piyapan039285 — APPROVED** [review]
well done

**Comment by earthpyy** [comment]
About three 1., please see [this link](https://github.com/DavidAnson/markdownlint/issues/45)

**Review by powerdefy — APPROVED** [review]

**Pull request merged**